### PR TITLE
Shared config through a WS

### PR DESCRIPTION
**Pull request by Mariocj89**
#171

### CODE REVIEW

**Comment by Mariocj89** [comment]
@dnaranjo89 please review, go over the commits as it is easier like that.

File: static/js/chat.js [hunk 1]
```diff
@@ -88,22 +83,25 @@
             });
         },
 
+        set messages(messages) {
```
Comment (dnaranjo89) [inline static/js/chat.js]: this function name has a space in it

File: static/js/chat.js [hunk 2]
```diff
@@ -56,11 +56,6 @@
                }
             });
 
-            // Auto refresh the chat against the server
-            (function start_auto_refresh () {
-                that.get_messages();
-                setTimeout(start_auto_refresh,5000);
```
Comment (dnaranjo89) [inline static/js/chat.js]: if this code is removed, what is making the chat refresh?

File: static/js/chat.js [hunk 3]
```diff
@@ -88,22 +83,25 @@
...
+            $chat.html("");
+            for (var i = 0, length = messages.length; i < length; i++) {
+                var element = messages[i];
+                var entry = this.formatChatEntry(element);
+                $chat.append(entry);
+            }
+        },
+
         // Get all the messages of a public draw and refresh the chat board
         get_messages: function (){
```
Comment (dnaranjo89) [inline static/js/chat.js]: Is this code still here for some reason? Seems that it's not used anymore

**Comment by dnaranjo89** [comment]
I didn't notice this before but, as there are two chats, there shouldn't be elements with id inside them (chat-board, chat-message-box, chat-send). Can you make them classes?

File: static/js/chat.js (hunk 1 above)
Comment (Mariocj89) [inline static/js/chat.js]: it is a setter

File: static/js/chat.js (hunk 3 above)
Comment (Mariocj89) [inline static/js/chat.js]: the normal draw uses it, we'll remove it when we remove the normal draw.

**Comment by Mariocj89** [comment]
about the changes in the chat plugin containing plugins I'd leave that for "the phase" 2 of refactoring it

**Pull request merged**